lib/transform_utils.js

```diff
@@ -32,8 +33,9 @@ var version_incrementer = function(new_version) {
           });
         }
       });
+      
 
-      if (_.has(ans, 'version') ||
+      if (_.has(ans, 'version') && Version.is_semantic_version(ans.version) && 
```

Comment:
This saves from not touching versions which are not in the top_types and which are not semantic else inner versions during recursive transform are change to latest ans version, example Version: 1 of type `children` in custom_embed will be changed to Version: 0.10.10